docs/static/releasenotes.asciidoc

```diff
@@ -0,0 +1,38 @@
+[[releasenotes]]
+== Release Notes
+
+This section summarizes the changes in the following releases:
+
+* <<logstash-5-5-0,Logstash 5.5.0>>
+
+[[logstash-5-5-0]]
+=== Logstash 5.5.0 Release Notes
+
+* Introduced a new feature called dead letter queues. When enabled, the dead letter queue (DLQ) 
+  will route events to a file based store when it can't be processed by certain plugins. Currently, 
+  the Elasticsearch Output is the only plugin that uses this feature to store events that cause a 
+  Elasticsearch mapping error.
+* Upgraded to jruby version 1.7.27.
```

Comment:
Not sure that we're ever consistent, but we should use correct capitalization: JRuby. :-)